### DISCUSSION TITLE
Modbus RTU Read_Coil  & Read_Status Example

### DISCUSSION THREAD

**Discussion by Thermelgy-Repo** [eModbus/eModbus#240] [Ideas]
Dear Team,

Thankq so much for your efforts to make this an amazing library. I have one suggestion to please include an example program to Read the Coil status (F01), Input Status (F02) so that it will be easy to understand the concept of reading using Coil datatype.

**Comment by Miq1** [comment]
Please have a look at the answer I gave in the issue you have opened in parallel [here](https://github.com/eModbus/eModbus/issues/241)
